app/assets/javascripts/helpers/profile_photo_uploader.es6

```diff
@@ -0,0 +1,250 @@
+// @license magnet:?xt=urn:btih:0b31508aeb0634b347b8270c7bee4d411b5d4109&dn=agpl-3.0.txt AGPL-v3-or-Later
+
+Diaspora.ProfilePhotoUploader = class {
+  /**
+   * Initializes a new instance of ProfilePhotoUploader
+   */
+  constructor() {
+    // get several elements we will use a few times
+    this.fileInput = document.querySelector("#file-upload");
+    this.picture = document.querySelector("#profile_photo_upload .avatar");
+    this.info = document.querySelector("#fileInfo");
+    this.cropContainer = document.querySelector(".crop-container");
+    this.spinner = document.querySelector("#file-upload-spinner");
+
+    /**
+     * Creates a button
+     * @param {string} icon - The entypo icon class.
+     * @param {function} onClick - Is called when button has been clicked.
+     */
+    this.createButton = (icon, onClick) =>
+      ($(`<button class="btn btn-default" type="button"><i class="entypo-${icon}"></i></button>`)
+        .on("click", onClick));
+
+    /**
+     * Shows a message using flash messages or alert for mobile.
+     * @param {string} type - The type of the message, e.g. "error" or "success".
+     * @param text - The text to display.
+     */
+    this.showMessage = (type, text) => { return app.flashMessages ? app.flashMessages[type](text) : alert(text); };
```

Comment:
I guess using implicit returns here is fine:

```js
this.showMessage = (type, text) => app.flashMessages ? app.flashMessages[type](text) : alert(text);
```

Reply:
Was my initial idea, but the active eslint rule [no-confusing-arrow](https://eslint.org/docs/rules/no-confusing-arrow) doesn't accept it.

Reply:
I... totally knew that! Disregard, then.

Reply:
on the other hand: I don't like this rule and see no confusion here. We could deactivate it or at least set `allowParens` to `true`. The implicit return looks so much nicer.

Reply:
I like, and understand the rule. However, with `allowParens`, it's fine by me. @svbergerem, any opinion?

Reply:
I just pushed an update and enabled `allowParens`. Looks good for me.